app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = [Person(human["name"], human["age"]) for human in people]
+
+    for human in people:
+        if "wife" in human and human["wife"]:
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif "husband" in human and human["husband"]:
+            Person.people[human["name"]].husband\
+                = Person.people[human["husband"]]
```

Comment:
```suggestion
        person = Person.people[human["name"]]
        if wife_name := human.get("wife"):
            person.wife = Person.people[wife_name]
        if husband_name := human.get("husband"):
            person.husband = Person.people[husband_name]
```

You can use the walrus operator to improve the code's readability.